pkg/neorpc/result/version.go

```diff
@@ -40,7 +40,9 @@ type (
 		ValidatorsCount             byte
 		InitialGasDistribution      fixedn.Fixed8
 		// Hardforks is the map of network hardforks with the enabling height.
-		Hardforks map[config.Hardfork]uint32
+		Hardforks        map[config.Hardfork]uint32
+		StandbyCommittee []string
```

Comment:
Why not a `[]crypto.PublicKey`?

Reply:
in config we have it as `[]string` https://github.com/nspcc-dev/neo-go/blob/5f745dff72e471304d33131506a3352becefcec5/pkg/config/protocol_config.go#L54

Reply:
Sure, but they're keys in their essence. I'm mostly concerned about client side, depending on scenario it might be more convenient to have public keys ready to use.